src/app/routines/exersiceModal/validationSchema.ts

```diff
@@ -0,0 +1,10 @@
+import * as Yup from 'yup';
+
+export const validationSchema = Yup.object().shape({
+  exerciseId: Yup.string().required('Exercise must be chosen'),
+  defaultSets: Yup.number().required('Sets is required'),
+  defaultReps: Yup.number().required('Reps is required'),
+  defaultWeight: Yup.number().required('Weight is required'),
+  defaultRestTime: Yup.string().required('Rest time is required'),
```

Comment:
it would be better to have rest time as number and have combo box right after it with selecting, seconds, minutes, hours.
Then combining those 2 fields in request and send to backend